source/type/data.cpp

```diff
@@ -854,6 +855,14 @@ void Data::clone_from(const Data& from) {
     write.join();
 }
 
+void Data::set_last_rt_data_loaded(const boost::posix_time::ptime& p) const{
```

Comment:
I might have miss something, but when is this function been called ?

Reply:
here: 
  - https://github.com/CanalTP/navitia/pull/2491/files#diff-da5424d87837c204fdedbd5ff51d9ed5R199 
  - https://github.com/CanalTP/navitia/pull/2491/files#diff-da5424d87837c204fdedbd5ff51d9ed5R208

Reply:
:see_no_evil: